packages/query/src/bitcoin/ordinals/brc20/brc20-tokens.query.ts

```diff
@@ -39,8 +39,8 @@ export function useGetBrc20TokensQuery({
   const query = useInfiniteQuery({
     queryKey: [QueryPrefixes.GetBrc20Tokens, currentNsBitcoinAddress, network.id],
     async queryFn({ pageParam }) {
-      const fromIndex: number = pageParam?.fromIndex ?? 0;
-      let addressesWithoutTokens = pageParam?.addressesWithoutTokens ?? 0;
+      const fromIndex = pageParam.fromIndex;
+      let addressesWithoutTokens = pageParam.addressesWithoutTokens;
```

Comment:
I know this isn't your code but maybe we could refactor this to not use `let`. Not in this PR but just flagging that using `let` can be confusing and harder to follow what's happening.  

So that we set:
```
const addressesWithoutTokens = pageParam.addressesWithoutTokens;

const brc20addressesWithoutTokens = brc20Tokens.filter(tokens => tokens.length === 0).length

...

return {
...
 addressesWithoutTokens: addressesWithoutTokens += brc20addressesWithoutTokens
}
```